tools/releases/dockerfiles/Dockerfile.base

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/distroless/base-debian11:debug-nonroot
```

Comment:
I'm not sure how this is going to work with entreprise and ubi base images.

Reply:
Not sure which aspect of "work" you mean, so I'll try to summarise aspects that I see:

- will distroless images be able to run our cointainers?
  - for enterprise - enterprise images are a copy of `kuma` images with different binaries, my gut feeling is everything should be fine
  - for UBI - there is no google/distroless for this, we can try using "UBI Micro" image

- can we have a base/static image?
  - for enterprise - our makefiles call kuma targets and the way it's structured now is that we have the same path leading to the Dockerfiles so that's why we have a copy of the dockerfiles. This PR does not change this approach so we would need a copy there as well and it should work the same way as it works now.
  - for UBI - ubi images have separate targets in enterprise makefiles, as mentioned above there is no distroless but we can have base image and replicate the approach taken here.

I think the most interesting question is:
- do we want to base enterprise images on top of kuma images (so have `FROM kumahq/base-debian11` in them) and if we do - how do we achieve that.

Reply:
Chat with Jakub he added the UBI stuff and will be able to provide details

Reply:
See enterprise PR to see how UBI is handled.